package.json

```diff
@@ -99,6 +100,7 @@
     "@reach/auto-id": "^0.6.1",
     "@reach/dialog": "^0.5.4",
     "@reduxjs/toolkit": "^1.4.0",
+    "@reload/dpl-design-system": "0.0.0-0ede2ac4e55c0a4ce4c960210dcf0a0ddb4fc7af",
```

Comment:
Do we have a clear strategy of which packages to use here?
Branch names is more human understandable, I guess.

Reply:
yes, that is true, but we need to be able to distinguish between packages of the same branch on each push. 
That's why we take the commit hash (i believe)

Reply:
mmm yeah. It is currently not very readable for a human eye. We could maybe add the branch name before the commit id